float32[] control_covariance
